vscode-dotnet-runtime-library/src/Acquisition/AcquisitionInvoker.ts

```diff
@@ -50,6 +51,7 @@ export class AcquisitionInvoker extends IAcquisitionInvoker {
                             this.eventStream.post(new DotnetOfflineFailure(offlineError, installContext.version));
                             reject(offlineError);
                         } else {
+                            error.message = `Installation failed, possibly due to a timeout: ${error.message}`;

```

Comment:
I'd imagine detecting a timeout failure would be valuable for another extension to understand.  Is there a way for us to detect that the underlying script timed out and then reject with an appropriate error (I'm making the big assumption that the error is not already understandable as a timeout)?

Reply:
Good idea, helps with telemetry too. See my latest update: a3774f7